code/data-pipeline/src/data_pipeline/loader.py

```diff
@@ -404,7 +410,7 @@ def load_agg_transform_flusurv(self, fips_mappings):
     return df_flusurv
 
 
-  def load_data(self, sources=None, hhs_kwargs={}):
+  def load_data(self, sources=None, flusurvnet_kwargs=None, hhs_kwargs=None, ilinet_kwargs=None):
```

Comment:
Responding to your comment on the related issue, yes...setting the default value to `None` is the pattern to use in place of using a mutable default like `{}` or `[]` 👍